diff -urN conserver-8.1.14/configure conserver-8.1.14-nate/configure
--- conserver-8.1.14/configure	2006-04-07 09:38:24.000000000 -0500
+++ conserver-8.1.14-nate/configure	2006-04-26 14:25:07.000000000 -0500
@@ -5847,6 +5847,9 @@
 	    OPENSSLCPPFLAGS=""
 	    OPENSSLLDFLAGS=""
 	fi
+	if test -e "/usr/kerberos/include/krb5.h"; then
+	    OPENSSLCPPFLAGS="$OPENSSLCPPFLAGS -I/usr/kerberos/include"
+	fi
 
 	oCPPFLAGS="$CPPFLAGS"
 	oLDFLAGS="$LDFLAGS"
diff -urN conserver-8.1.14/configure.in conserver-8.1.14-nate/configure.in
--- conserver-8.1.14/configure.in	2006-04-07 09:30:22.000000000 -0500
+++ conserver-8.1.14-nate/configure.in	2006-04-26 14:24:53.000000000 -0500
@@ -456,6 +456,9 @@
 	    OPENSSLCPPFLAGS=""
 	    OPENSSLLDFLAGS=""
 	fi
+	if test -e "/usr/kerberos/include/krb5.h"; then
+	    OPENSSLCPPFLAGS="$OPENSSLCPPFLAGS -I/usr/kerberos/include"
+	fi
 
 	oCPPFLAGS="$CPPFLAGS"
 	oLDFLAGS="$LDFLAGS"
